### ISSUE TITLE
Rule Proposal: Warn if custom functions are used for browser.wait 

### ISSUE THREAD

**Issue by alecxe** [alecxe/eslint-plugin-protractor#35]
It is quite rare there is the need to write a custom function for browser.wait(). Much more often there is an existing Expected Condition for the problem. Sometimes, users just don't know that there is already a condition covering the use case.

Example of a violation:

``` js
browser.wait(function () {
    return browser.getCurrentUrl().then(function (url) {
        return url.indexOf("word") >= 0;
    });
}, 5000);
```

Correct usage:

``` js
browser.wait(protractor.ExpectedConditions.urlContains("word"), 5000);
```

Should probably turn the rule off by default.

**Comment by alecxe** [comment]
Does not seem like anyone would actually turn this rule on. Closing.
